src/status_im2/contexts/syncing/scan_sync_code/style.cljs

```diff
@@ -77,29 +77,37 @@
    :top      (- (+ (:y viewfinder) (:height viewfinder)) flash-button-size flash-button-spacing)
    :right    (+ screen-padding flash-button-spacing)})
 
-(defn border
-  [border1 border2 corner]
-  (assoc {:border-color colors/white
-          :width        78
-          :height       78}
-         border1
-         2
-         border2
-         2
-         corner
-         16))
-
-(defn border-tip
-  [top bottom right left]
-  {:background-color colors/white
-   :position         :absolute
-   :top              top
-   :bottom           bottom
-   :right            right
-   :left             left
-   :height           2
-   :width            2
-   :border-radius    2})
+(defn- get-border
+  [border-vertical border-horizontal corner-radius]
+  {:border-color     colors/white
+   :width            78
+   :height           78
+   border-vertical   2
+   border-horizontal 2
+   corner-radius     16})
+
+(def white-border
+  (let [base-tip {:background-color colors/white
+                  :position         :absolute
+                  :height           1.9
```

Comment:
it might be worth adding a comment for use of 1.9 here.

Reply:
Done :+1: